docs/DevOps.adoc

```diff
@@ -34,14 +34,14 @@ When a pull request has changes to asciidoc files, you can use https://www.netli
 
 Here are the steps to create a new release.
 
-.  Update the version number in link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp.java`].
+.  Update the version number in webLink:{repoURL}/src/main/java/seedu/description/MainApp.java[`MainApp.java`].
```

Comment:
I think not all `link`s should be changed to `webLink` and `address` should be changed to `algobase`